src/main/java/seedu/address/model/description/Description.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.model.description;
+
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Description {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Description should only contain alphanumeric characters and spaces, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
This validation regex is for names. Should this be dropped?

Reply:
Do you think it is okay to drop?

Reply:
I think we should just keep the consistency of checking regex, but for description we just allow "" since it is an optional field for the user?